### PR TITLE
fix(input): change password field preview icon to actual svg so it can be filled white when active

### LINKED ISSUE #4804: Firefox accounts - change to white password show icon when button pressed
**Affected versions**
* 75.0b10, 76.0a1 (2020-03-29)

**Affected platforms**
* macOS 10.15, Windows 10, Ubuntu 18.04;

**Steps to reproduce**
1. Launch Firefox, click to signin with any valid account and input the email address;
2. Click on the show password button after inputing anything in that string;
* after signing in;
3. Click on the Firefox account button;
4. Click on the Change password;
5. Input any password;
6. Click on the show password button.

**Expected result**
* icon is visible;

**Actual result**
* black icon in blue button;

**Enhancement suggestion**
* step 2 and 6:
* icon can be changed to white, such as buttons  from Change/Add/Enable;

**Regression range**
* not a regression, most likely a scenario that was overlooked;

**Additional notes**
* as requested in bug [1625899](https://bugzilla.mozilla.org/show_bug.cgi?id=1625899)

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1523)
┆Issue Number: FXA-1523

### PR DESCRIPTION
**Pull request by jodyheavener**
Closes #4804

Makes this:

Screen Shot 2020-04-15 at 11 10 26 AM

Look like this:

Screen Shot 2020-04-15 at 11 06 35 AM

I'm not convinced it looks great white, but it looks better than before I guess.

### CODE REVIEW

**Comment by jodyheavener** [comment]
I'm so torn about how this looks :/

**Comment by LZoog** [comment]
I think @ryanfeeley was considering this at some point but we left it black.

Should definitely ask @pwalm for confirmation that we want this.

File: packages/fxa-content-server/app/scripts/templates/partial/show-password.mustache [hunk 1]
```diff
@@ -1,2 +1,4 @@
 <input id="{{ id }}" type="checkbox" class="show-password" aria-controls="{{ targetId }}" tabindex="-1" data-novalue data-synchronize-show="{{ synchronizeShow }}"/>
-<label for="{{ id }}" class="show-password-label" title="{{#t}}Show password{{/t}}"><img alt="{{#t}}Show password{{/t}}" src="data:image/svg+xml;base64,PHN2ZyBkYXRhLW5hbWU9IkxheWVyIDIiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgd2lkdGg9IjU1IiBoZWlnaHQ9IjQ2IiB2aWV3Qm94PSItMTYgLTEzIDQ3IDM4Ij48cGF0aCBkPSJNMTYgNS43QTguMzIgOC4zMiAwIDAgMCA4IDAgOC4zMiA4LjMyIDAgMCAwIC4wNSA1LjdhLjg2Ljg2IDAgMCAwIDAgLjU5QTguMzEgOC4zMSAwIDAgMCA4IDEyYTguMzIgOC4zMiAwIDAgMCA4LTUuNy44OC44OCAwIDAgMCAwLS42ek04LjUgM0ExLjUgMS41IDAgMSAxIDcgNC41IDEuNSAxLjUgMCAwIDEgOC41IDN6TTggMTBhNi4zMyA2LjMzIDAgMCAxLTUuOTQtNCA2LjA4IDYuMDggMCAwIDEgMi41LTNBNCA0IDAgMCAwIDQgNWE0IDQgMCAwIDAgOCAwIDQgNCAwIDAgMC0uNTYtMiA2IDYgMCAwIDEgMi41IDNBNi4zMyA2LjMzIDAgMCAxIDggMTB6Ii8+PC9zdmc+Cg=="></label>
+<label for="{{ id }}" class="show-password-label" title="{{#t}}Show password{{/t}}">
```
Comment (LZoog) [inline packages/fxa-content-server/app/scripts/templates/partial/show-password.mustache:2]: This probably needs an `aria-label` saying "Show password", I don't think it'll read just "title".

**Comment by pwalm** [comment]
I'd say go with the white for now, and we can revisit this when we revise our sign-up flow later this year.

**Pull request merged**